src/main/java/org/mitre/dsmiley/httpproxy/ProxyServlet.java

```diff
@@ -453,7 +477,8 @@ protected void copyProxyCookie(HttpServletRequest servletRequest,
       Cookie servletCookie = new Cookie(proxyCookieName, cookie.getValue());
       servletCookie.setComment(cookie.getComment());
       servletCookie.setMaxAge((int) cookie.getMaxAge());
-      servletCookie.setPath(path); //set to the path of the proxy servlet
+      servletCookie.setPath(cookie.getPath());
```

Comment:
It's been too long since I looked at this, what does this change do?  I suspect this would break what I changed for cookies earlier but not sure.  If you add some unit tests that would make it much clearer what you're doing and make sure my cookie changes still work.